### PR TITLE
CI improvements

### PR DESCRIPTION
**Pull request by Krzmbrzl**

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
In order for this here to be buildable, it has to be rebased against master, after #10 has been merged

File: src/Opus.hpp [hunk 1]
```diff
@@ -38,15 +38,15 @@ template< typename T > class OpusBase {
 	std::unique_ptr< T, Destructor > m_ctx;
 };
 
-class Opus::Decoder::P : public OpusBase< ::OpusDecoder > {
+class Opus::Decoder::P : public OpusBase<::OpusDecoder > {
```
Comment (davidebeatrici) [inline src/Opus.hpp:41]: A bug in `clang-format`?

Comment (Krzmbrzl) [reply src/Opus.hpp:41]: Yeah, that's a quirk of clang-format v10

Comment (davidebeatrici) [reply src/Opus.hpp:41]: I wonder what version I had when all the extra spaces were added...

Comment (Krzmbrzl) [reply src/Opus.hpp:41]: I believe this issue is fixed from version 11 onwards, but I am not quite sure.
However, I would argue that we should stay consistent with our setup in main Mumble in that regard.

Comment (davidebeatrici) [reply src/Opus.hpp:41]: Yes, absolutely.

**Pull request merged**